README.md

```diff
@@ -518,6 +518,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [Prismic](https://www.prismic.io/) — Headless CMS. Content management interface with fully hosted and scalable API. The Community Plan provides unlimited API calls, documents, custom types, assets, and locales to one user. Everything that you need for your next project. Bigger free plans are available for Open Content/Open Source projects.
   * [Sanity.io](https://www.sanity.io/) - Platform for structured content with an open-source editing environment and a real-time hosted data store. Unlimited projects. Unlimited admin users, three non-admin users, two datasets, 500K API CDN requests, 10GB bandwidth, and 5GB assets included for free per project.
   * [sensenet](https://sensenet.com) - API-first headless CMS providing enterprise-grade solutions for businesses of all sizes. The Developer plan provides three users, 500 content items, three built-in roles, 25+5 content types, fully accessible REST API, document preview generation, and Office Online editing.
+  * [Strapi](https://strapi.io/) - Strapi is the leading open-source headless CMS. 100% JavaScript and fully customizable.
```

Comment:
It’s not a SaaS.

Reply:
It's not a PaaS either, this is not a list for this kind of software.

Previously also rejected as not suited for this list https://github.com/ripienaar/free-for-dev/pull/1757

Thanks for the contribution.